### PR TITLE
bump/1.1.0

### PR DESCRIPTION
**Pull request by aboedo**
updated changelog, version number and releasing instructions

### Requirements: 
- [x] #29

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
 - Make a branch `bump/x.x.x`
 - Update purchases-android version in `android/common/build.gradle`
-- Update CHANGELOG.md
+- Update purchases-ios pod version in `PurchasesHybridCommon.podspec`, `ios/PurchasesHybridCommon/Podfile`, and `ios/PurchasesHybridCommon/PurchasdsHybridCommon.xcworkspace` (open this last one with Xcode)
+- Update `CHANGELOG.md`
```
Comment (aboedo) [inline RELEASING.md:6]: we'll eventually wanna automate this, but my thinking was to first have it working and in use

**Comment by aboedo** [comment]
hold removed

File: RELEASING.md (hunk 1 above)
Comment (vegaro) [reply RELEASING.md:6]: Yeah this works

File: RELEASING.md [hunk 2]
```diff
@@ -2,7 +2,9 @@
 
 - Make a branch `bump/x.x.x`
 - Update purchases-android version in `android/common/build.gradle`
```
Comment (vegaro) [inline RELEASING.md:4]: We are missing the bump in this file :)

Comment (aboedo) [reply RELEASING.md:4]: oops yep

Comment (aboedo) [reply RELEASING.md:4]: @vegaro updated

**Pull request merged**